.github/workflows/pylint.yml

```diff
@@ -0,0 +1,45 @@
+
+
+name: Pylint
+
+on:
+  workflow_dispatch:
```

Comment:
This is fine for development, but since you are running Pylint on every push, maybe you can eventually drop it.